packages/cf-deploy-config-writer/src/cf-writer/base-config.ts

```diff
@@ -53,7 +53,7 @@ async function addRoutingConfig(config: CFBaseConfig, fs: Editor): Promise<void>
     if (config.routerType === RouterModuleType.Standard) {
         await addStandaloneRouter(config, mtaConfigInstance, fs);
     } else {
-        await mtaConfigInstance.addRoutingModules(true);
+        await mtaConfigInstance.addRoutingModules(true, false);
```

Comment:
Thanks @longieirl 
PR is good and changes are clear.
A minor comment - would be nice to pass params as options here...
`options: { isManagedApp?: boolean; doCleanUp?: boolean } `